spec/lib/bandiera/logging_audit_log_spec.rb

```diff
@@ -1,33 +1,62 @@
 require 'spec_helper'
 
 RSpec.describe Bandiera::LoggingAuditLog do
-  let(:logger) { instance_double('Logger') }
+  let(:db) { Bandiera::Db.connect }
   let(:audit_context) { Bandiera::AnonymousAuditContext.new }
-  subject { Bandiera::LoggingAuditLog.new(logger) }
+  subject { Bandiera::LoggingAuditLog.new(db) }
 
   it_behaves_like 'an audit log'
 
   describe '#record' do
-    it 'records the audit message to the logger' do
-      expect(logger).to receive(:log).with('AUDIT [<anonymous>] add foodstuff (name: burger)')
+    it 'records the audit message to the audit log' do
+      expect(db[:audit_records]).to be_empty
 
       subject.record(audit_context, :add, :foodstuff, name: 'burger')
+
+      audit_record = db[:audit_records].first
+      expect(audit_record).to_not be_nil
+      expect(audit_record[:user]).to eq('<anonymous>')
+      expect(audit_record[:action]).to eq('add')
+      expect(audit_record[:object]).to eq('foodstuff')
+      expect(JSON.parse(audit_record[:params]).symbolize_keys).to eq({ name: 'burger' })
+    end
+
+    it 'sets the timestamp to the current time' do
+      expect(db[:audit_records]).to be_empty
+
+      start_time = Time.now
+      subject.record(audit_context, :add, :foodstuff, name: 'burger')
+      end_time = Time.now
+
+      audit_record = db[:audit_records].first
+      expect(audit_record).to_not be_nil
+      expect(audit_record[:timestamp]).to be_between(start_time, end_time)
```

Comment:
You can use a gem called `timecop` for time based testing in Ruby (https://github.com/travisjeffery/timecop).  This would allow you to assert that it is the exact time and not just approximate.